documentation/docs/40-best-practices/05-performance.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Performance
+---
+
+Out of the box, SvelteKit does a lot of work to make your applications as performant as possible:
+
+- Code-splitting, so that only the code you need for the current page is loaded
+- Asset preloading, so that 'waterfalls' (of files requesting other files) are prevented
+- File hashing, so that your assets can be cached forever
+- Request coalescing, so that data fetched from separate server `load` functions is grouped into a single HTTP request
+- Parallel loading, so that separate universal `load` functions fetch data simultaneously
+- Data inlining, so that requests made with `fetch` during server rendering can be replayed in the browser without issuing a new request
+- Conservative invalidation, so that `load` functions are only re-run when necessary
+- Prerendering (configurable on a per-route basis, if necessary) so that pages without dynamic data can be served instantaneously
+- Link preloading, so that data and code requirements for a client-side navigation are eagerly anticipated
```

Comment:
If I'm remembering correctly, we don't do this by default, so we could probably tweak it a bit:
```suggestion
- Support for link preloading, so that data and code requirements for a client-side navigation are eagerly anticipated
```

Reply:
We do do it by default, insofar as the `<body>` element in a newly created project has `data-sveltekit-preload-data="hover"`

Reply:
Ah, I'd forgotten that. Thanks for the pointer! I'll resolve this comment